### PR TITLE
Task/more tests

### PR DESCRIPTION
**Pull request by andreattid**

### CODE REVIEW

File: tests/features/sync_flow.feature [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+      | ATT_MAPPING           | []                |
+      | ATT_INTERACTION_TYPE  | synchronous       |
+      | ATT_TIMEOUT           | 120               |
+    And a valid token is retrieved for user "<SERVICE_ADMIN>" and password "<SERVICE_PWD>"
+    And a device "<DEVICE_ID>" of entity_type "<ENTITY_TYPE>" should be provisioned for service and subservice
+      | attribute            | value          |
+      | TOKEN                | <TOKEN>        |
+      | ATT_INTERACTION_TYPE | synchronous    |
+      | ATT_GEOLOCATION      | <ATT_LOCATION> |
+
```
Comment (XavierVal) [inline tests/features/sync_flow.feature]: empty

File: tests/features/sync_flow.feature [hunk 2]
```diff
@@ -0,0 +1,164 @@
+Feature: Complete path E2E functionality Sync-Async / Sync-Sync
+  In order to check if a blackbutton that can order a product
+  In a Service servicea1a and Subservice testpizza
```
Comment (XavierVal) [inline tests/features/sync_flow.feature]: Service name should match with the names in the tests

File: tests/features/sync_flow.feature [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+      | attribute             | value                   |
+      | PROTOCOL              | <PROTOCOL>              |
+      | DEVICE_ID             | <DEVICE_ID>             |
+      | ENTITY_TYPE           | <ENTITY_TYPE>           |
+      | SERVICE_NAME          | <SERVICE_NAME>          |
+      | SUBSERVICE_NAME       | <SUBSERVICE_NAME>       |
+      | SERVICE_USER_NAME     | <SERVICE_USER_NAME>     |
+      | SERVICE_USER_PASSWORD | <SERVICE_USER_PASSWORD> |
+      | TOKEN                 | <TOKEN>                 |
+
```
Comment (XavierVal) [inline tests/features/sync_flow.feature]: remove empty line

File: tests/features/sync_flow.feature [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+      | PROTOCOL              | TT_BLACKBUTTON  |
+      | DEVICE_ID             | <DEVICE_ID>     |
+      | ENTITY_TYPE           | <ENTITY_TYPE>   |
+      | SERVICE_NAME          | <SERVICE>       |
+      | SUBSERVICE_NAME       | <SERVICEPATH>   |
+      | SERVICE_USER_NAME     | <SERVICE_ADMIN> |
+      | SERVICE_USER_PASSWORD | <SERVICE_PWD>   |
+      | TOKEN                 | <TOKEN>         |
+    Then device "<DEVICE_ID>" should be listed under service and subservice
+    #And the button "<DEVICE_ID>" pressed in mode "synchronous" the IOTA should not receive the request "<BT_REQUEST>"
```
Comment (XavierVal) [inline tests/features/sync_flow.feature]: unused line?

Comment (andreattid) [inline tests/features/sync_flow.feature]: To be used in further commits. For now, response to failured processing of messages exits with a timeout after pending for a few seconds, and real results can be only seen in log files.

File: tests/features/sync_flow.feature (hunk 3 above)
Comment (andreattid) [inline tests/features/sync_flow.feature]: Fixed in 96b280d

File: tests/features/sync_flow.feature (hunk 2 above)
Comment (andreattid) [inline tests/features/sync_flow.feature]: Fixed in 96b280d

File: tests/features/sync_flow.feature (hunk 1 above)
Comment (andreattid) [inline tests/features/sync_flow.feature]: Fixed in 96b280d

**Comment by XavierVal** [comment]
LGTM

**Pull request merged**